src/fmu/dataio/_definitions.py

```diff
@@ -83,16 +83,13 @@ class FmuContext(str, Enum):
     Use a Enum class for fmu_context entries.
 
     The different entries will impact where data is exported:
-    REALIZATION = "To realization-N/iter_M/share"
-    CASE = "To casename/share, but will also work on project disk"
-    PREPROCESSED = "To share/preprocessed; from interactive runs but re-used later"
-    NON_FMU = "Not ran in a FMU setting, e.g. interactive RMS"
-
+    REALIZATION: To realization-N/iter_M/share
+    CASE: To casename/share, but will also work on project disk
+    NON_FMU: Not ran in a FMU setting, e.g. interactive RMS.
     """
 
     REALIZATION = "realization"
```

Comment:
Not sure if relevant, but in the `ExportAggregatedData` class the context is set to "iteration". So that should possibly also be on this list, for clarity. It must at least be valid in the schema.

https://github.com/equinor/fmu-dataio/blob/9a08f4e2fa8af6db211a334384b2fd958d6f19fb/src/fmu/dataio/aggregation.py#L232

Reply:
Nice catch :) added 👍